### PR TITLE
Fix engagement route

### PR DESCRIPTION
**Pull request by laurieboyes**
Bit meaty I'm afraid but much clearer if you look at the individual commits

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -66,7 +66,7 @@ run-local:
 	export PORT=5454 \
 	export GRAPHENEDB_URL=http://${LOCAL_USER}:${LOCAL_PASSWORD}@localhost:7474; \
 	export GRAPHENEDB_HISTORY_URL=http://${LOCAL_USER}:${LOCAL_PASSWORD}@localhost:7474; \
-	export REDIS_URL=http://no:auth@localhost:6379; \
+	export REDIS_URL=http://localhost:6379; \
```
Comment (keirog) [inline Makefile]: ☔

**Comment by phamann** [comment]
👍 LGTM. Thanks for fixing

File: Makefile (hunk 1 above)
Comment (laurieboyes) [inline Makefile]: Yep. Will do a separate PR fixing this and updating the build
